components/Counter.js

```diff
@@ -0,0 +1,61 @@
+var Counter = React.createClass({
+    getInitialState: function() {
+        console.log("getInitialState: ustawienie stan początkowego");
```

Comment:
console.log should be used when you're debugging your code. in production code all console.logs should be removed